app/models/article.rb

```diff
@@ -0,0 +1,3 @@
+class Article < ApplicationRecord
+  validates_presence_of :title, :body, :author
```

Comment:
We had a discussion about author in one of the review sessions. Why is it present on this iteration? There is no mention about the author in this story on PT